content/documentation/pages/3-stream-developer-guides/2-streams/1-standalone-stream-sample.md

```diff
@@ -75,11 +75,11 @@ In this step, we create the `UsageDetailSender` source.
 
 <!--Kafka-->
 
-You can either [directly download the project generated by the Spring Initialzr](https://start.spring.io/#!type=maven-project&language=java&platformVersion=2.4.3.RELEASE&packaging=jar&jvmVersion=1.8&groupId=io.spring.dataflow.sample&artifactId=usage-detail-sender-kafka&name=usage-detail-sender-kafka&description=Demo%20project%20for%20Spring%20Boot&packageName=io.spring.dataflow.sample.usagedetailsender&dependencies=cloud-stream,actuator,kafka)
+You can either [directly download the project generated by the Spring Initialzr](https://start.spring.io/#!type=maven-project&language=java&platformVersion=2.4.12&packaging=jar&jvmVersion=1.8&groupId=io.spring.dataflow.sample&artifactId=usage-detail-sender-kafka&name=usage-detail-sender&description=Demo%20project%20for%20Spring%20Boot&packageName=io.spring.dataflow.sample.usagedetailsender&dependencies=cloud-stream,actuator,kafka)
```

Comment:
All of the Initializr links were updated w/ the following changes:

- 2.4.3.RELEASE -> 2.4.12 as the former causes a big warning popup about 2.4.12 being the latest 2.4x release.
- Remove the binder specific suffix from the generated app in order to match the class names in the example test code which does not have a `(Kafka|Rabbit)Application` suffix.